### PR TITLE
Vertical list json

### PR DESCRIPTION
**Pull request by johnHackworth**
Adds a new verticals list, based on google AdWords'

### CODE REVIEW

File: packages/data-stores/src/verticals/resolvers.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { receiveVerticals } from './actions';
  * API response.
  */
 export function* getVerticals() {
-	const url = 'https://public-api.wordpress.com/wpcom/v2/verticals';
+	const url = '/calypso/page-templates/vertical-list.json';
```
Comment (sirreal) [inline packages/data-stores/src/verticals/resolvers.ts:23]: There's no reason to use a resolver for a static value. The selector can return the value.

Comment (sirreal) [reply packages/data-stores/src/verticals/resolvers.ts:23]: 😕 This isn't a static value… we're adding a file to calypso and then using a network request to get a JSON response from it via a static URL?

This doesn't seem quite right. If we'd like to provide the contents of a local JSON file, let's require the JSON file and return it.

What's being done here right now seems off.

Comment (simison) [reply packages/data-stores/src/verticals/resolvers.ts:23]: @sirreal This PR is just for a demo to see how the format of the new verticals works in onboarding. The idea is to serve the content from an API.

**Comment by simison** [comment]
Test link:

https://hash-097204a1ec2c3a4b7dee253f48df862f7a87284c.calypso.live/new?fresh

BTW the default animated suggestions aren't on the new list:

Old

Screenshot 2020-04-29 at 11 40 16

New

Screenshot 2020-04-29 at 11 41 18

Not sure if that matters (cc @dubielzyk). The good news is that "photo" brings more options now. :-)

File: packages/data-stores/src/verticals/resolvers.ts (hunk 1 above)
Comment (sirreal) [reply packages/data-stores/src/verticals/resolvers.ts:23]: Sounds good. Let's avoid temporarily publishing this at a public URL, just require and return it.

**Comment by razvanpapadopol** [comment]
We can close this for now. See https://github.com/Automattic/wp-calypso/issues/37630

**Pull request closed without merging**